connection/keepAlive.js

```diff
@@ -0,0 +1,33 @@
+const KeepAlive = require('../packets/clientbound/keepAlive.js')
+
+class KeepAliveHandler {
```

Comment:
moved all the keep alive stuff into a separate file. Was previously spread over a bunch of different files and was impossible to track